doc/users/interactive.rst

```diff
@@ -280,25 +270,24 @@ If ``ipympl`` is installed use the magic:
 
 to select and enable it.
 
-If you only need to use the classic notebook you can use
+If you only need to use the classic notebook, you can use
 
 .. sourcecode:: ipython
 
   %matplotlib notebook
 
-which uses the `.backend_nbagg` backend which ships with Matplotlib.
-However nbagg does not work in Jupyter Lab.
+which uses the `.backend_nbagg` backend which is installed by Matplotlib; 
+however, nbagg does not work in Jupyter Lab.
 
-GUIs + jupyter
+GUIs + Jupyter
 ~~~~~~~~~~~~~~
 
-If you are running your jupyter kernel locally you can use one of the
-GUI backends.  The process running your kernel will show a GUI window
-on your desktop adjacent to your web browser.  However if you move
-that notebook to a remote server the kernel will try to open the GUI
-window on *that* computer.  Unless you have arranged to forward the
-xserver back to your desktop, you not be able to see or interact with
-the figure (if it does not raise an exception outright).
+You can also use one of the non-ipympl GUI backends in a Jupyter Notebook. 
```

Comment:
```suggestion
You can also use one of the non-``ipympl`` GUI backends in a Jupyter Notebook. 
```

Reply:
I am also not sure I would call `ipympl ` a GUI backend